src/borg/remote.py

```diff
@@ -78,6 +126,22 @@ def __init__(self, restrict_to_paths, append_only):
         self.repository = None
         self.restrict_to_paths = restrict_to_paths
         self.append_only = append_only
+        self.client_version = (1, 1, 0)
```

Comment:
Currently this uses 3-tuples for versioning. But setuptools_scm gives strings. Do we need something special for parsing them into nicely comparable tuples? How would we like to handle prereleases?

We could just pass the version, but then we are in trouble if we ever want to change the version format. These tend to be not purely technical in nature.